rain_api_core/view_util.py

```diff
@@ -187,5 +194,38 @@ def make_set_cookie_headers_jwt(payload, expdate='', cookie_domain=''):
     return headers
 
 
+def is_jwt_blacklisted(decoded_jwt):
+    urs_user_id = decoded_jwt["urs-user-id"]
+
+    if decoded_jwt["urs-user-id"] in JWT_BLACKLIST["blacklist"]:
```

Comment:
Perhaps use the `urs_user_id` variable here?

Reply:
Inside the if statement?

Reply:
If that is done, the code will fail if the user isn't in the blacklist because of the logging message at the end of the function.

Reply:
On line 198, you define `urs_user_id = decoded_jwt["urs-user-id"]`.
Then on line 200, you use `decoded_jwt["urs-user-id"]`.  Wouldn't it make more sense to write it as:

```
if urs_user_id in JWT_BLACKLIST["blacklist"]:
```

Reply:
Nevermind, I see what you're saying